### PR TITLE
BZ1785123 internal Ingress Controller for AWS restricted networks

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1785123

@jianlinliu, will you PTAL?

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@jianlinliu, will you please take another look? I think that the request to clarify how to make the ingress controller private makes sense in this context, even if the private cluster method exists. I think that making the step optional is the right thing to do, since it's not actually mandatory.

File: modules/installation-generate-aws-user-infra-install-config.adoc [hunk 1]
```diff
@@ -109,6 +109,14 @@ imageContentSources:
 ----
 +
 Use the `imageContentSources` section from the output of the command to mirror the repository or the values that you used when you mirrored the content from the media that you brought into your restricted network.
+
+.. Optional: Set the publishing strategy to `Internal`
++
+----
```
Comment (adellape) [inline modules/installation-generate-aws-user-infra-install-config.adoc:115]: Would adding `[source,yaml]` to this (and any other relevant code blocks in this module) be appropriate?

**Review by adellape — COMMENTED** [review]
Just one musing.

**Pull request merged**